### ISSUE TITLE
Elevated water tank

### ISSUE THREAD

**Issue by olgunk83** [DualSPHysics/DualSPHysics#28]
Hello, firstly thank you for the work of design sphysics. My thesis is about elevated water tanks. I installed Design sphysics on freecad.But I couldn't get the pressures on the tank wall.I would be very happy if you can review the file I have attached.
I wish you good work.
Thanks...
https://drive.google.com/open?id=1llwZFRNeFD2UkgXX0kVJIOe_-7Qg_YOP

**Comment by zlotny** [comment]
Hi there. 

First of all, take in mind you're submitting an issue for DualSPHysics (the command line suite), not DesignSPHysics (the GUI). That would be here: https://github.com/DualSPHysics/DesignSPHysics/issues

Secondly. I analysed a bit the case you attached and... it's really strange. The gravity works from top to bottom (that is, like FreeCAD represents, from +z to -z). You were designing the case sideways.

The tank that you added is at a whopping 30km! from the origin (0,0,0). where did you find that STL file? If you made it please center it at (0,0,0) before saving/exporting it to prevent this. FreeCAD will just add the STL as is, without understanding its void space.

The world in DesignSPHysics is completely 3D, even if you want to do a 2D case, in which basically you "slice" that 3D space. The fillbox that you added is near the origin (0,0,0) as it should be, but obviously is 30km away of your tank.

![image](https://user-images.githubusercontent.com/6155245/69710778-47c53480-1100-11ea-8239-fa6beff04069.png)

So, in short. Try to fix your STL, the tank should be at (or near) 0,0,0 in whatever 3d modeling software you're using. Also, don't make water out of an STL, just the fillbox is fine, as external shapes will only render as faces (only the particles will be at the faces of the object, not filling it).

I advise you to first try to do a simple case (like a cube of water falling into a floor, or a dam break, something like that) to start understanding how it works and how FreeCAD works.

Thanks for using DesignSPHysics, if you have further issues with it please post them in the issues in its repository: https://github.com/DualSPHysics/DesignSPHysics/issues
